tests/features/authentication.feature

```diff
@@ -18,13 +18,10 @@ Feature: User authentication
     But I should not see the text "Log out"
     And I should not see the text "My account"
 
-  Scenario Outline: Anonymous user can access public pages
+  Scenario: Anonymous user can access public pages
     Given I am not logged in
-    Then I visit "<path>"
-
-    Examples:
-      | path                           |
-      | rdf_entity/http%3A\\drupal.org |
+    When I go to the homepage of the "Überwaldean Land Eels" collection
+    Then I should see the heading "Überwaldean Land Eels"
```

Comment:
Isn't this already tested? Why is this here in the authentication test? This should be in the collection test suite.